db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+class Race(models.Model):
+    RACE_NAME_CHOICES = (
+        ("elf", "Elf"),
+        ("dwarf", "Dwarf"),
+        ("human", "Human"),
+        ("ork", "Ork")
+    )
+    name = models.CharField(
+        choices=RACE_NAME_CHOICES,
+        unique=True,
+        max_length=255
+    )
```

Comment:
what will happen when you will need to add more choices ?

Reply:
We can simply add the necessary race name to choices, if we'll get an update. In our task has written, that we have 4 races:
![image](https://github.com/mate-academy/py-game-models/assets/122704517/2a8abfe5-f06d-4b15-ac85-0b93a09f8e71)
I think it would be better to add the available races to the choices to avoid duplicate races that would only differ by 1 letter due to a missclick and add new races to choices if it'll be necessary.

Reply:
`I think it would be better to add the available races to the choices to avoid duplicate races that would only differ by 1 letter due to a missclick and add new races to choices if it'll be necessary.`
agree